### PR TITLE
add bulgarian tranlsation

### PR DESCRIPTION
**Pull request by mkkDr2010**
Here the .ts file
[bg.zip](https://github.com/sddm/sddm/files/5672466/bg.zip)

### CODE REVIEW

File: src/daemon/XorgDisplayServer.cpp [hunk 1]
```diff
@@ -136,106 +141,97 @@ namespace SDDM {
...
-            displayNumber.remove(displayNumber.size() -1, 1); // trim trailing whitespace
-            m_display = QString::fromLocal8Bit(displayNumber);
+        // close the other side of pipe in our process, otherwise reading
+        // from it may stuck even X server exit.
+        close(pipeFds[1]);
+
+        QFile readPipe;
+
+        if (!readPipe.open(pipeFds[0], QIODevice::ReadOnly)) {
+            qCritical("Failed to open pipe to start X Server");
```
Comment (mathieujobin) [inline src/daemon/XorgDisplayServer.cpp:200]: why so much changes for adding bulgarian translation?

**Comment by plfiorini** [comment]
There are many unrelated commits

**Review by plfiorini — CHANGES_REQUESTED** [review]
Many unrelated commits, please clean the PR

**Comment by mathieujobin** [comment]
this is a common pattern when you mix git merges with rebase.

I am guessing he had many branches that are not his included in his branch, which got rebased unto master. but he did a git pull and merged the rebase commits.

the solution here is to rebase everything one more time with `master`.

but OH ! did `develop` and `master` went out of sync?

**Comment by mathieujobin** [comment]
looks like he opened a PR with develop onto master, probably a mistake. 
his translation isn't even part of this PR 

lets close this BEFORE you update master with develop, otherwise, github will automatically show it as merged.

**Review by mathieujobin — CHANGES_REQUESTED** [review]
please close and open fresh one

**Pull request closed without merging**